af_process_log/views/af_process_log.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="view_process_log_tree" model="ir.ui.view">
+        <field name="name">af.process.log.tree</field>
+        <field name="model">af.process.log</field>
+        <field name="arch" type="xml">
+            <tree string="AF Process Log">
+                <field name="process"/>
+                <field name="eventid"/>
+                <field name="create_date"/>
+                <field name="write_date"/>
+                <field name="step"/>
+                <field name="error_message"/>
+                <field name="status"/>
+                <field name="no_tries"/>
+                <field name="objectid"/>
```

Comment:
maybe add the info_1/2/3 fields in the view here and in the form view?

Reply:
I have added the three info-fields as suggested